### PR TITLE
template update: task w/ user-assigned identity

### PR DESCRIPTION
**Pull request by dlepow**
@northtyphoon @huanwu - Updated template for task with user-assigned identity, based on one of Huangli's examples. I plan to point to this template from a new article about running build tasks from ARM templates.

### CODE REVIEW

**Comment by huanwu** [comment]
@dlepow  Do you need update readme file for the change?

**Comment by dlepow** [comment]
@huanwu - Good point, I updated readme.md as well

File: docs/tasks/run-as-deployment/quickdockerbuildwithidentity/README.md [hunk 1]
```diff
@@ -23,20 +30,41 @@ az identity create \
...
+customregistryName=$(az acr show -n mycustomreg --query loginServer --output tsv)
+
+#Get resource ID of the identity
+resourceId==$(az identity show --resource-group mytaskrunrg --name myquickdockerbuildrunwithidentity --query id --output tsv)
+
+#Get client ID of the identity
+clientId=$(az identity show --resource-group mytaskrunrg --name myquickdockerbuildrunwithidentity --query clientId --output tsv)
 
+az group deployment create --resource-group "mytaskrunrg" --template-file azuredeploy.json \
+  --parameters azuredeploy.parameters.json \
```
Comment (northtyphoon) [inline docs/tasks/run-as-deployment/quickdockerbuildwithidentity/README.md:62]: @danlep@microsoft.com @huanwu@microsoft.com I see `az group deployment` is deprecated. Can you update it to `az deployment`? 


Group
    az group deployment : Manage Azure Resource Manager deployments.
        This command group has been deprecated and will be removed in a future release. Use
        'deployment group' instead.

Comment (dlepow) [reply docs/tasks/run-as-deployment/quickdockerbuildwithidentity/README.md:62]: @northtyphoon @huanwu - Updated command to `az deployment group` throughout.

**Pull request merged**